projects/igniteui-angular/src/lib/grids/cell.component.ts

```diff
@@ -821,9 +821,11 @@ export class IgxGridCellComponent implements OnInit, OnChanges, OnDestroy {
      * @hidden
      * @internal
      */
-    public activate(event: FocusEvent | KeyboardEvent) {
+    public activate(event: FocusEvent | KeyboardEvent, position) {
```

Comment:
activate method is called in the scope of the cell component. We do not have a case where we call this method on one cell in order to activate another. So why is necessary to pass position arguments since in the code below you can just pass this.rowIndex and this.visibleColumnIndex as params to the setActiveNode method.
 this.grid.navigation.setActiveNode({ row: this.rowIndex, column: this.visibleColumnIndex });